library/java/org/chromium/net/impl/CronetUrlRequest.java

```diff
@@ -403,11 +403,11 @@ private void onResponseHeaders(ResponseHeaders responseHeaders, boolean lastCall
       return;
     }
     final List<Map.Entry<String, String>> headerList = new ArrayList<>();
-    String selectedTransport = "http/1.1"; // TODO(carloseltuerto) looks dubious
```

Comment:
Is this the ALPN issue? Should we leave it with a TODO?

Reply:
I added a todo further down, line 431

``` // TODO(https://github.com/envoyproxy/envoy-mobile/issues/1546) negotiated protocol```

For HTTP1, the logic here is fine: the expected value is ```"unknown"``` - that's what the original test is looking for.

For HTTP2/3, I don't have tests yet, as I need to make TLS to work for tests.